crates/experimental-api/src/exe/map.rs

```diff
@@ -29,7 +39,51 @@ impl TryFrom<StarkFelt> for gen::Felt {
 impl TryFrom<gen::GetClassResult> for ContractClass {
     type Error = Error;
 
-    fn try_from(_value: gen::GetClassResult) -> Result<Self, Self::Error> {
-        unimplemented!() // TODO: implement necessary mapping
+    fn try_from(value: gen::GetClassResult) -> Result<Self, Self::Error> {
+        let json = serde_json::to_string(&value)?;
+        Ok(match value {
+            gen::GetClassResult::ContractClass(_) => {
+                let class = ContractClassV1::try_from_json_string(&json)?;
+                ContractClass::V1(class)
+            }
+            gen::GetClassResult::DeprecatedContractClass(class) => {
+                let class = build_contract_class(class)?;
+                ContractClass::V0(class)
+            }
+        })
     }
 }
+
+fn build_contract_class(
+    class: DeprecatedContractClass,
+) -> Result<ContractClassV0, Error> {
+    let program = decode_program(class.program.as_ref())?;
+
+    let mut class = serde_json::to_value(class)?;
+    class["program"] = serde_json::from_str(&program)?;
+    let json = serde_json::to_string_pretty(&class)?;
```

Comment:
We should probably use `serde_json::to_string` here to avoid unnecessary allocations.

Maybe something like this to keep the debug readability?
```
#[cfg(debug_assertions)]
let json = serde_json::to_string_pretty(&class)?;
#[cfg(not(debug_assertions))]
let json = serde_json::to_string(&class)?;
```

Reply:
`to_string` is more than enough here